spec/models/classification_spec.rb

```diff
@@ -585,6 +585,20 @@
         expect(classification.name).to eq(new_name)
         expect(classification.tag.name).to eq(Classification.name2tag(new_name, category))
       end
+
+      it "doesn't update the tag when something other than the name changes" do
+        classification.reload
+        expect do
+          classification.update_attributes(:description => new_name)
+        end.to match_query_limit_of(2)
```

Comment:
match_query_limit_of(2) looks confusing to me.
If we are checking that `tags` table was not updated in `Classification#before_save` callback, may be we can check that `tag.update_attributes` was not called

Reply:
thnx - fixed